src/pages/en/guides/markdown-content.md

```diff
@@ -1,153 +1,73 @@
 
-```js
-import autolinkHeadings from 'remark-autolink-headings';
 
-// astro.config.mjs
-export default {
-  markdownOptions: {
-    render: [
-      '@astrojs/markdown-remark',
-      {
-        remarkPlugins: [[autolinkHeadings, { behavior: 'prepend' }]],
-      },
-    ],
-  },
-};
-```
 
-### Syntax Highlighting
+```markdown
+---
+# Example: src/pages/index.md
+title: Hello, World
+---
+# Hi there!
 
-Astro comes with built-in support for [Prism](https://prismjs.com/) and [Shiki](https://shiki.matsu.io/). By default, Prism is enabled. You can modify this behavior by updating the `@astrojs/markdown-remark` options:
+This is your first markdown page. It probably isn't styled much, although
+Markdown does support **bold** and *italics.* 
 
-```js
-// astro.config.mjs
-export default {
-  markdownOptions: {
-    render: [
-      '@astrojs/markdown-remark',
-      {
-        // Pick a syntax highlighter. Can be 'prism' (default), 'shiki' or false to disable any highlighting.
-        syntaxHighlight: 'prism',
-        // If you are using shiki, here you can define a global theme and
-        // add custom languages.
-        shikiConfig: {
-          theme: 'github-dark',
-          langs: [],
-          wrap: false,
-        },
-      },
-    ],
-  },
-};
+To learn more about adding a layout to your page, read the next section on **Markdown Layouts.**
 ```
 
-You can read more about custom Shiki [themes](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) and [languages](https://github.com/shikijs/shiki/blob/main/docs/languages.md#supporting-your-own-languages-with-shiki).
-
-## Markdown Pages
-
-Astro treats any `.md` files inside of the `/src/pages` directory as pages. These files can contain frontmatter, but are otherwise processed as plain markdown files and do not support components. If you're looking to embed rich components in your markdown, take a look at the [Markdown Component](#astros-markdown-component) section.
-
-### Layouts
+### Markdown Layouts
 
-Markdown pages have a special frontmatter property for `layout`. This defines the relative path to an `.astro` component which should wrap your Markdown content, for example a [Layout](/en/core-concepts/layouts) component. All other frontmatter properties defined in your `.md` page will be exposed to the component as properties of the `content` prop. The rendered Markdown content is placed into the default `<slot />` element.
+Markdown pages have a special frontmatter property for `layout` that defines the relative path to an `.astro` [layout component](/en/core-concepts/layouts). This component will wrap your Markdown content, providing a page shell and any other included page template elements. 
```

Comment:
So, is it “an `.astro` component” or “a `.astro` component”? I read “dot-astro” so would assume “a dot-astro component” but this is interesting!

Reply:
Oh, I've always "said" "an astro component."  I also say "an md file" in my head, too.

Ok, this is one to figure out, as it'll matter site-wide.

Reply:
Since it's only once in this page, and it's got a link right there to layout components (thereby also indicating official Astro-y), I'm going to just change it to "Astro layout component" !!